build

```diff
@@ -1,10 +1,29 @@
 #!/bin/bash
+#
+# Build a new Python Docker image.
+#
+# Usage:
+#   ./build [--gpu] [--use-cache]
+#
+# Options:
+#   --gpu: Build an image with GPU support.
+#   --use-cache: Use layer cache when building a new image.
+# 
 set -e
+set -x
+
+CACHE_FLAG="--no-cache"
+IMAGE_TAG="kaggle/python-build"
+DOCKERFILE="Dockerfile"
+
+if [[ "$1" == "--gpu" ]]; then
```

Comment:
if `argv[1]` is `--use-cache` and `argv[2]` is `--gpu`, both will be ignored right?  The usage is correct in that regard, but it still comes unexpected.  Consider using `getopts` or at least mentioning that in the usage explicitly.

Reply:
`getopts` doesn't support long options. I implemented it by hand. I also made it nicer to use by supporting short flags and displaying help. I also updated the other scripts even though they only have one argument to also display help and support the short flags.